test/functional/normalization/eventrouter_test.go

```diff
@@ -0,0 +1,128 @@
+package normalization
+
+import (
+	"encoding/json"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+	"github.com/openshift/cluster-logging-operator/test/matchers"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/reference"
+	"strings"
+	"time"
+)
+
+const timestamp string = "1985-10-21T09:00:00.00000+00:00"
+
+var (
+	templateForAnyKubernetes = types.Kubernetes{
+		ContainerName:     "*",
+		PodName:           "*",
+		NamespaceName:     "*",
+		NamespaceID:       "*",
+		OrphanedNamespace: "*",
+	}
+	templateForAnyCollector = types.PipelineMetadata{Collector: types.Collector{
+		Ipaddr4:    "*",
+		Inputname:  "*",
+		Name:       "*",
+		Version:    "*",
+		ReceivedAt: time.Time{},
+	},
+	}
+	nanoTime, _ = time.Parse(time.RFC3339Nano, timestamp)
```

Comment:
Exactly like `timestamp` this can move into the Describe context (you will need to pass `nanoTime`  to the template creator but this is ok )

Reply:
fixed